docs/notebooks/index.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+title: Notebook
+nav_order: 38
+has_children: false
+---
+
+# Kibana Notebook
+
+A Kibana notebook is an interface that lets you easily create and share live log data reports.
```

Comment:
one of the key differentiators is 
- ability to view multi-timeline reports

Reply:
Does multi-timeline mean adding two visualizations paragraphs with different date ranges for comparative analysis?

Reply:
Please refer to What's new